src/api/index.ts

```diff
@@ -1,7 +1,31 @@
-export * from "./MainApi"
-export * from "./RetryType"
-export * from "./Article"
-export * from "./Tag"
-export * from "./Lecture"
-export * from "./MainApi"
-export * from "./User"
+import { articles } from "./articles"
+import { auth } from "./auth"
+import { tags } from "./tags"
+import { timetable } from "./timetable"
+import { user } from "./user"
+export * from "./httpClient"
```

Comment:
imho non serve esporre il client nell'API, fare l'import da `"api/HttpClient"` da più l'idea di qualcosa che non va fatto spesso ahah
```suggestion
```

Reply:
giusto

Reply:
Allora esporto le interface in HttpClient.ts in modo indipendente (RetryType etc...)

Reply:
Espongo comunque RetryType, AuthType, RequestOptions, va bene?